### PR TITLE
Fix Shifts Filter (date selection, pre-selected types)

### PR DESCRIPTION
**Pull request by JanLuca**

### CODE REVIEW

File: includes/model/ShiftsFilter.php [hunk 1]
```diff
@@ -12,8 +12,8 @@ class ShiftsFilter {
   /**
    * How long can the time interval be?
    */
-  const MAX_DURATION = 86400;
- // one day
+  const MAX_DURATION = 1036800;
+ // twolve days
```
Comment (fabianfreyer) [inline includes/model/ShiftsFilter.php:16]: if typo, please use `twölve`.

File: includes/pages/user_shifts.php [hunk 2]
```diff
@@ -43,18 +43,21 @@ function update_ShiftsFilter_timerange(ShiftsFilter $shiftsFilter, $days) {
   $start_time = $shiftsFilter->getStartTime();
   if ($start_time == null) {
     $start_time = time();
+    if($start_time < 1495238400) {
```
Comment (fabianfreyer) [inline includes/pages/user_shifts.php:46]: would be really nice to somehow construct these timestamps from a date. It seems [PHP is able to do that](http://php.net/manual/en/function.strtotime.php).

File: includes/pages/user_shifts.php [hunk 3]
```diff
@@ -43,18 +43,21 @@ function update_ShiftsFilter_timerange(ShiftsFilter $shiftsFilter, $days) {
...
+      $start_time = 1495238400; // 20.05.2017 00:00
+    }
   }
   
   $end_time = $shiftsFilter->getEndTime();
   if ($end_time == null) {
-    $end_time = $start_time + 24 * 60 * 60;
+    $end_time = $start_time + 12 * 24 * 60 * 60;
```
Comment (fabianfreyer) [inline includes/pages/user_shifts.php:53]: Does this mean the default shift length is 12 days?!?

File: includes/model/ShiftsFilter.php [hunk 4]
```diff
@@ -48,12 +48,13 @@ public function __construct($user_shifts_admin, $rooms, $types) {
...
     $this->filled = [
-        ShiftsFilter::FILLED_FREE 
+        ShiftsFilter::FILLED_FREE,
+        ShiftsFilter::FILLED_FILLED
     ];
     
-    if ($user_shifts_admin) {
-      $this->filled[] = ShiftsFilter::FILLED_FILLED;
-    }
+//    if ($user_shifts_admin) {
```
Comment (fabianfreyer) [inline includes/model/ShiftsFilter.php:55]: just delete this, there's `git blame` and `git diff`.

File: includes/pages/user_shifts.php (hunk 2 above)
Comment (JanLuca) [reply includes/pages/user_shifts.php:46]: Yeah, I was in a train without internet connection so this was the dirty but working solution. This is a dirty fix, anyway, so I think this is OK enough.

File: includes/pages/user_shifts.php (hunk 3 above)
Comment (JanLuca) [reply includes/pages/user_shifts.php:53]: No, this is the default end time for the shifts view. By default all shifts between 2017-05-20 and 2017-05-30 are shown now.

**Pull request merged**